modules/apim_gateway/manifests/params.pp

```diff
@@ -16,51 +16,37 @@
 
 # Class apim_gateway::params
 # This class includes all the necessary parameters.
-class apim_gateway::params {
-  $user = 'wso2carbon'
-  $user_group = 'wso2'
-  $product = 'wso2am'
-  $product_version = '2.6.0'
-  $service_name = 'wso2am'
-  $profile = 'gateway'
-  # $local_ip = $::ipaddress
-
-  # JDK Distributions
-  if $::osfamily == 'redhat' {
-    $lib_dir = "/usr/lib64/wso2"
-  }
-  elsif $::osfamily == 'debian' {
-    $lib_dir = "/usr/lib/wso2"
-  }
-  $jdk_name = 'amazon-corretto-8.202.08.2-linux-x64'
-  $java_home = "${lib_dir}/${jdk_name}"
+class apim_gateway::params inherits apim_common::params {
 
   $start_script_template = 'bin/wso2server.sh'
+  $jvmxms = '256m'
+  $jvmxmx = '1024m'
+
+  $template_list = [
+    'repository/conf/api-manager.xml',
+    'repository/conf/datasources/master-datasources.xml',
+    'repository/conf/carbon.xml',
+    'repository/conf/user-mgt.xml',
+    'repository/conf/axis2/axis2.xml',
+  ]
+
+  # Define file list
+  $file_list = []
```

Comment:
Why are defining these empty file lists?

Reply:
The user can define files they need to add to the setup. The files get added here: https://github.com/wso2/puppet-apim/pull/114/files#diff-f887e469de4b8a03342d0df119a077afR35